flik/linesearch/interpolations.py

```diff
@@ -93,19 +93,19 @@ def phi_(alpha):
         """The derivative of the previously defined function phi."""
         return grad(var + alpha * direction).dot(direction)
 
-
     # If interval_begin or interval_end are not minimizers, then the minimizer lies in the interval
     # ]interval_begin, interval_end[
-    d_1 = phi_(alpha_previous) + phi_(alpha_current) \
-          - 3 * (phi(alpha_previous) - phi(alpha_current)) / (alpha_previous - alpha_current)
-
-    d_2 = np.sign(alpha_current - alpha_previous) \
-          * np.sqrt(d_1 ** 2 - phi_(alpha_previous) * phi_(alpha_current))
+    d_1 = phi_(alpha_previous) + phi_(alpha_current) - 3 * (phi(alpha_previous) -
+                                                            phi(alpha_current)) / (alpha_previous -
+                                                                                   alpha_current)
 
-    alpha_next = alpha_current - \
-                 (alpha_current - alpha_previous) * (phi_(alpha_current) + d_2 - d_1) / \
-                 (phi_(alpha_current) - phi_(alpha_previous) + 2 * d_2)
+    d_2 = np.sign(alpha_current - alpha_previous) * np.sqrt(d_1 ** 2 - phi_(alpha_previous) *
+                                                            phi_(alpha_current))
 
+    alpha_next = alpha_current - (alpha_current - alpha_previous) * (phi_(alpha_current) + d_2 -
+                                                                     d_1) / (phi_(alpha_current) -
+                                                                             phi_(alpha_previous) +
+                                                                             2 * d_2)
```

Comment:
I really don't agree on enforcing "E127 continuation line over-indented for visual indent" on this one. The formulas are completely illegible.